### DISCUSSION TITLE
Roboform Import doesn't work

### DISCUSSION THREAD

**Discussion by 33-viper-33** [dani-garcia/vaultwarden#3553] [Q&A] [answered]
I just set up vaultwarden in docker using vaultwarden/server:latest and attempted to import roboform from .csv.  Unfortunately, nothing happens.  After a google search, it looks like this is a known issue.  https://github.com/bitwarden/web/issues/419  It appears that this current pull is using version 2023.3 instead of the latest 2023.4x.  

Can this pull be updated or is there a way to fix it manually?

**Comment by BlackDex** [comment, marked as answer]
Use the testing tagged docker images, they already have the latest web-vault.

**Reply by 33-viper-33** [reply]
I actually pulled an old version 1.26.0, which fixed it.  Same idea, but I went backwards vs forward in builds. I'll upgrade again shortly to the latest.

Thank you so much!  Appreciate all your work!

**Reply by 33-viper-33** [reply]
for any other novices out there like myself, when you pull the image, use "vaultwarden/server:1.26.0" to go backwards.  Once you do your roboform import, upgrade the image using "vaultwarden/server:latest".

**Comment by Thomas-HN72** [comment]
Hi,
so I am new on this vaultwarden. I installed a docker container also with the latest image. I am not able to import my roboform data! I stored in folders - and the folders are visible.
Of course the workaround: version 1.26.0 installing doesn't work for me too.
What is the problem?
Thanks
Thomas
